src/Agent/NewRelic/Agent/Core/Configuration/DefaultConfiguration.cs

```diff
@@ -876,12 +876,88 @@ private bool IsDistributedTracingEnabled()
+            }
 
-        public RemoteParentSampledBehavior RemoteParentNotSampledBehavior => EnvironmentOverrides(_localConfiguration.distributedTracing.sampler.remoteParentNotSampled.ToString(), "NEW_RELIC_DISTRIBUTED_TRACING_SAMPLER_REMOTE_PARENT_NOT_SAMPLED").ToRemoteParentSampledBehavior();
+            // use local config if no env var override
+            var samplerItem = samplerLevel switch
+            {
+                SamplerLevel.Root => _localConfiguration.distributedTracing.sampler.root.Item,
+                SamplerLevel.RemoteParentSampled => _localConfiguration.distributedTracing.sampler.remoteParentSampled.Item,
+                SamplerLevel.RemoteParentNotSampled => _localConfiguration.distributedTracing.sampler.remoteParentNotSampled.Item,
+                _ => throw new ArgumentOutOfRangeException(nameof(samplerLevel), samplerLevel, null)
+            };
 
-        public bool TraceIdRatioBasedSamplingEnabled => _localConfiguration.distributedTracing.root.traceIdRatioBased.ratioSpecified;
-        public float? TraceIdRatioBasedSamplingRatio => TraceIdRatioBasedSamplingEnabled ? _localConfiguration.distributedTracing.root.traceIdRatioBased.ratio : null;
+            //if samplerLevel is one of the remote parent levels and samplerItem is null, we need to check the legacy attributes
+            if (samplerLevel != SamplerLevel.Root && samplerItem == null)
+            {
+                var remoteParentSampledBehaviorType = samplerLevel == SamplerLevel.RemoteParentSampled ?
+                    _localConfiguration.distributedTracing.sampler.remoteParentSampled1 :
+                    _localConfiguration.distributedTracing.sampler.remoteParentNotSampled1;
+
+                if (remoteParentSampledBehaviorType != RemoteParentSampledBehaviorType.@default)
+                {
+                    Log.Warn($"Using deprecated configuration for {samplerLevel}. Please use the new distributed tracing sampler configuration.");
+                    return remoteParentSampledBehaviorType.ToRemoteParentSamplerType();
+                }
+            }
+
+            return MapSamplerItem(samplerItem);
+        }
+
+        private static SamplerType MapSamplerItem(object samplerItem) =>
+            samplerItem switch
+            {
+                null => SamplerType.Default,          // not configured
+                DefaultSamplerType => SamplerType.Default,
+                AlwaysOnSamplerType => SamplerType.AlwaysOn,
+                AlwaysOffSamplerType => SamplerType.AlwaysOff,
+                TraceIdRatioSamplerType => SamplerType.TraceIdRatioBased,
+                _ => throw new ArgumentOutOfRangeException(nameof(samplerItem), samplerItem, "Unknown sampler type in configuration.")
+            };
+
+        private float? TraceIdSamplerRatio(SamplerLevel samplerLevel)
+        {
+            // TODO: do we need to support environment variable override for these? Naming will be difficult.
```

Comment:
I think customers are going to want that, but I think we can simplify it to the cases that make the most sense and just offer:
* Ratio for root sampling
* Honor the remote parent sampled flag (do we not have this already?)

We can do this as a separate work item, though